election-day/Pratster95/election_day.go

```diff
@@ -0,0 +1,42 @@
+package electionday
+
+import "fmt"
+
+// NewVoteCounter returns a new vote counter with
+// a given number of inital votes.
+func NewVoteCounter(initialVotes int) *int {
+	var counter *int
```

Comment:
You do not need the extra variable `counter`. You can return the address of `initialVotes` like `return &initialVotes`.